src/java.desktop/share/classes/javax/swing/JTable.java

```diff
@@ -1164,7 +1164,7 @@ public Color getGridColor() {
      * @see     #setShowHorizontalLines
      */
     @BeanProperty(description
-            = "The color used to draw the grid lines.")
+            = "Whether grid lines should be drawn in around the cells.")
```

Comment:
What about
"Whether grid lines should be drawn around the cells."
or 
"Whether grid lines should be drawn around cells."?